packages/client3/components/design-system/input/input.tsx

```diff
@@ -103,60 +109,109 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
-            decoration ? "pr-8" : null,
-            textSize === "sm"
-              ? "py-1.5 px-3"
-              : textSize === "md"
-              ? "py-2 px-3"
-              : textSize === "lg"
-              ? "px-4 py-3"
-              : textSize === "xl"
-              ? "px-5 py-4"
-              : null,
-            inputClassName
-          )}
-          ref={ref}
-          name={name}
-          id={_id}
-          type={type}
-          disabled={disabled}
-          autoComplete={autoComplete}
-          {...rest}
-        />
-        {typeof decoration === "string" ? (
-          <Icon
-            name={decoration as IconNameType}
-            className="absolute right-3.5 top-1/2 -translate-y-1/2"
+              isError ? "border-state-error" : null,
+
+              disabled && "opacity-50",
+              decoration ? "pr-8" : null,
+              textSize === "sm"
+                ? "py-1.5 px-3"
+                : textSize === "md"
+                ? "py-2 px-3"
+                : textSize === "lg"
+                ? "px-4 py-3"
+                : textSize === "xl"
+                ? "px-5 py-4"
+                : null,
+              inputClassName
+            )}
+            ref={ref}
+            onClick={() => {
+              setFocus(true);
+            }}
+            onBlur={(event) => {
+              onBlur && onBlur(event);
+              setFocus(false);
+            }}
+            name={name}
+            id={_id}
+            type={type}
+            disabled={disabled}
+            autoComplete={autoComplete}
+            {...rest}
           />
-        ) : decoration ? (
-          <div className="absolute right-3.5 top-1/2 -translate-y-1/2">
-            {decoration}
+          <div
+            className={clsx(
+              isMoneyInput ? "absolute right-14 top-1/2 -translate-y-1/2" : null
+            )}
+          >
+            <span className=" text-dark-light-4 sm:text-sm">
+              {(() => {
+                if (isMoneyInput) return <span>USDC</span>;
+              })()}
+            </span>
           </div>
-        ) : null}
+          {typeof decoration === "string" ? (
+            <Icon
+              name={decoration as IconNameType}
+              className="absolute right-3.5 top-1/2 -translate-y-1/2"
+            />
+          ) : decoration ? (
+            <div className="absolute right-3.5 top-1/2 -translate-y-1/2">
+              {decoration}
+            </div>
+          ) : null}
+        </div>
       </div>
       {helperText || _errorMessage ? (
-        <></>
-      ) : // <HelperText
-      //   className={clsx(
-      //     isError ? "text-state-error" : "text-dark-80",
-      //     "mt-1 text-sm leading-none"
-      //   )}
-      // >
-      //   {_errorMessage ? _errorMessage : helperText}
-      // </HelperText>
-      null}
+        <Caption
+          className={clsx(
+            isError ? "text-state-error" : "text-dark-50",
```

Comment:
I think is `text-dark-80`

Reply:
![image](https://user-images.githubusercontent.com/98773781/189358619-590b3a50-5501-436c-8a4e-b89a7c67dc66.png)
```
/* dark/grey50 */

color: #909090;
```

Reply:
My bad